src/fidesops/migrations/versions/7abe778b7082_update_fideslang_data_categories.py

```diff
@@ -0,0 +1,217 @@
+    "user.provided.identifiable.contact.email": "user.contact.email",
+    "user.provided.identifiable.contact.phone_number": "user.contact.phone_number",
+    "user.provided.identifiable.credentials": "user.credentials",
+    "user.provided.identifiable.credentials.biometric_credentials": "user.credentials.biometric_credentials",
+    "user.provided.identifiable.credentials.password": "user.credentials.password",
+    "user.provided.identifiable.date_of_birth": "user.date_of_birth",
+    "user.provided.identifiable.financial": "user.financial",
+    "user.provided.identifiable.financial.account_number": "user.financial.account_number",
+    "user.provided.identifiable.gender": "user.gender",
+    "user.provided.identifiable.genetic": "user.genetic",
+    "user.provided.identifiable.government_id": "user.government_id",
+    "user.provided.identifiable.government_id.drivers_license_number": "user.government_id.drivers_license_number",
+    "user.provided.identifiable.government_id.national_identification_number": "user.government_id.national_identification_number",
+    "user.provided.identifiable.government_id.passport_number": "user.government_id.passport_number",
+    "user.provided.identifiable.health_and_medical": "user.health_and_medical",
+    "user.provided.identifiable.job_title": "user.job_title",
+    "user.provided.identifiable.name": "user.name",
+    "user.provided.identifiable.non_specific_age": "user.non_specific_age",
+    "user.provided.identifiable.political_opinion": "user.political_opinion",
+    "user.provided.identifiable.race": "user.race",
+    "user.provided.identifiable.religious_belief": "user.religious_belief",
+    "user.provided.identifiable.sexual_orientation": "user.sexual_orientation",
+    "user.provided.identifiable.workplace": "user.provided.identifiable.workplace",
+}
+
+DATA_MAP_DOWNGRADE = {v: k for k, v in DATA_MAP_UPGRADE.items()}
+
+
+def is_removed_downgrade(privacy_key: str) -> bool:
+    removed = (
+        "user.contact.address",
+        "user.contact.address.number",
+        "user.financial.payment_information",
+        "user.name.first",
+        "user.name.last",
+    )
+
+    return privacy_key in removed
+
+
+def is_removed_upgrade(privacy_key: str) -> bool:
+    removed = (
+        "account",
+        "account.contact",
+        "account.contact.city",
+        "account.contact.country",
+        "account.contact.email",
+        "account.contact.phone_number",
+        "account.contact.postal_code",
+        "account.contact.state",
+        "account.contact.street",
+        "account.payment.financial_account_number",
+        "user.derived",
+        "user.derived.identifiable",
+        "user.derived.nonidentifiable",
+        "user.derived.nonidentifiable.sensor",
+        "user.provided",
+        "user.provided.identifiable",
+        "user.provided.nonidentifiable",
+    )
+
+    return privacy_key in removed
```

Comment:
Once removed in an upgrade, a downgrade will not be able to add these back because there will be no way to know where they should go. I don't think there is anything we can do about this.